config/config.php

```diff
@@ -222,14 +221,21 @@
 xG0LjsK2vZOV4ZHT3tZPHGUV1Nrpfj5LIuCyPgn0w+XqTBm+AuF1W5MsgtDq0GOo
 /l4U06AGoCaWHiYAKXbHGg==',
 			],
-			// For PHP 5.4 users
-			'0' => [
+			'101' => [
 				'latest' => '10.0.6',
 				'internalVersion' => '9.1.6.1',
 				'downloadUrl' => 'https://download.nextcloud.com/server/prereleases/nextcloud-10.0.6.zip',
 				'web' => 'https://docs.nextcloud.org/server/10/admin_manual/maintenance/manual_upgrade.html',
 				'minPHPVersion' => '5.4',
 			],
+			'102' => [
+				'latest' => '11.0.6',
+				'internalVersion' => '11.0.6.1',
+				'downloadUrl' => 'https://nextcloud.com/blog/update-your-php-version',
```

Comment:
Needs to be updated to the one from https://github.com/nextcloud/nextcloud.com/issues/686

Reply:
https://nextcloud.com/outdated-php-5-4-5-5/ <- that is the link